src/components/cookie-consent/cookie-consent.tsx

```diff
@@ -44,6 +46,11 @@ export class CookieConsent {
 		this.checkExcludedPaths();
 	}
 
+	componentDidLoad() {
+		const focusTrap = createFocusTrap(this.componentRef);
+		setTimeout(() => focusTrap.activate());
```

Comment:
Volgens mij dient deze focus trap er niet te zijn wanneer men de 'vriendelijke' cookieconsent aanroept (`"nonBlocking": true`).

Reply:
Ik heb het in orde gebracht en getest met de nonBlocking property.